### PR TITLE
Updated compiler version to 1.0.46

### PR DESCRIPTION
**Pull request by tvinzz**

### CODE REVIEW

File: filters/ThirdParty/filter_217_PolskiFiltrCookies/diff.txt [hunk 1]
```diff
@@ -623,4 +545,464 @@ filmboxlive.com##.mobox-wrapper:style(overflow: auto !important;)
 ! Rule "galopuje.pl,sklepbiker.pl##.modal-open:style(padding-right: 0 !important; overflow: auto !important;)" converted to: galopuje.pl,sklepbiker.pl#$#.modal-open { padding-right: 0 !important; overflow: auto !important; }
 galopuje.pl,sklepbiker.pl##.modal-open:style(padding-right: 0 !important; overflow: auto !important;)
 ! Rule "powiatslubicki.pl##.modal-open:style(overflow: auto !important;)" converted to: powiatslubicki.pl#$#.modal-open { overflow: auto !important; }
-powiatslubicki.pl##.modal-open:style(overflow: auto !important;)
\ No newline at end of file
+powiatslubicki.pl##.modal-open:style(overflow: auto !important;)
+! 4lomza.pl#$#body.polityka-cookies { margin-top: 0 !important; } is excluded by: #$#
```
Comment (krystian3w) [inline filters/ThirdParty/filter_217_PolskiFiltrCookies/diff.txt]: Why exclude CSS?

Comment (Alex-302) [reply filters/ThirdParty/filter_217_PolskiFiltrCookies/diff.txt]: @tvinzz please change trust-level

Comment (krystian3w) [reply filters/ThirdParty/filter_217_PolskiFiltrCookies/diff.txt]: anyway I revert filters to `##*:style()`

Comment (tvinzz) [reply filters/ThirdParty/filter_217_PolskiFiltrCookies/diff.txt]: Added `high` trust level for `217_PolskiFilterCookies`

**Comment by krystian3w** [comment]
Possible recover CSS also for:

* https://github.com/AdguardTeam/FiltersRegistry/blob/master/filters/ThirdParty/filter_247_PolishRssFilters/diff.txt
* https://github.com/AdguardTeam/FiltersRegistry/blob/master/filters/ThirdParty/filter_237_PolishAnnoyance/diff.txt
* https://github.com/AdguardTeam/FiltersRegistry/blob/master/filters/ThirdParty/filter_221_PolskiFiltrSocial/diff.txt

**Pull request merged**